job-dsls/jobs/scrclr_scan_seed_job.groovy

```diff
@@ -0,0 +1,26 @@
+job('srcclr_scan_seed_job') {
+    description('Scan job, which generates scanning jobs for upstream projects')
+    parameters {
+        stringParam('REPO_FILE_URL','https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/master/script/repository-list.txt','URL of the repository-list.txt file')
+        stringParam('KIE_JENKINS_SCRIPTS_REPO', 'https://github.com/kiegroup/kie-jenkins-scripts', '')
+        stringParam('KIE_JENKINS_SCRIPTS_BRANCH', 'master', '')
+        stringParam('JOB_PATH', '', '')
+    }
+    scm {
+        git {
+            remote {
+                name('origin')
+                url("${KIE_JENKINS_SCRIPTS_REPO}")
```

Comment:
I think this needs not to be evaluated in the seed job, but just passed to Jenkins (otherwise the tests fail because the parameter is not set in the tests):
```suggestion
                url('$KIE_JENKINS_SCRIPTS_REPO')
```